README.md

```diff
@@ -1,7 +1,7 @@
 # Halmak Keyboard Layout
 
 This is an AI designed keyboard layout that was built within the
-[keyboard-gentics](https://github.com/MadRabbit/keyboard-genetics) project.
+[keyboard-genetics][] project.
```

Comment:
What are those extra square brackets on the end for? you don't need those.

Reply:
If you scroll to the bottom of the changes, I think this is a way for you to specify that the text is a link (e.g. "keyboard-genetics") but you specify where it links to at the bottom of the file

Reply:
You don't need extra brackets for [that]

[that]: https://ddg.gg

Standard markdown is just:

```
You don't need extra brackets for [that]

[that]: https://ddg.gg
```

Reply:
TIL! Thank you for sharing 🙏🏻

Reply:
```suggestion
[keyboard-genetics] project.
```